web/tests/test_rems.py

```diff
@@ -47,9 +113,25 @@ def test_rems_handler_duplicate(client, user_vip, user_data_steward):
     assert len(accesses) == 1
 
 
-def test_rems_handler_no_expiration(client, user_vip, user_data_steward):
+def test_rems_handler_no_expiration(client, user_vip, user_data_steward, mocker):
     resource_id = "TEST-2-5591E3-1"
     email = "john.doe@uni.lu"
+
+    def mock_get_external_user_info(self, oidc_id):
```

Comment:
It feels to me like this whole command could go outside the tests, to avoid repeating the definition.

The only thing that differs between the calls are the first name, last name, and email of the mock user. Could they be set as the method argument?

Reply:
done